derive/src/lib.rs

```diff
@@ -175,19 +175,30 @@
 //!
 //! Special rules can be called within the grammar. They are:
 //!
-//! * `whitespace` - gets run between rules and sub-rules
-//! * `comment` - gets run between rules and sub-rules
-//! * `any` - matches exactly one `char`
-//! * `soi` - (start-of-input) matches only when a `Parser` is still at the starting position
-//! * `eoi` - (end-of-input) matches only when a `Parser` has reached its end
-//! * `pop` - pops a string from the stack and matches it
-//! * `peek` - peeks a string from the stack and matches it
-//! * `drop` - drops the top of the stack (fails to match if the stack is empty)
-//!
-//! `whitespace` and `comment` should be defined manually if needed. All other rules cannot be
+//! * `WHITESPACE` - runs between rules and sub-rules
```

Comment:
I would convert this into two tables. First, the special rules, then the shorthands. This can be done in a separate PR, however.